ui/app/components/modal-form/policy-template.hbs

```diff
@@ -31,15 +31,14 @@
     {{#if (eq this.policy.policyType "acl")}}
       <p>
         ACL Policies are written in Hashicorp Configuration Language (
-        <DocLink @host="https://github.com/hashicorp/hcl">HCL</DocLink>
+        <ExternalLink @href="https://github.com/hashicorp/hcl">HCL</ExternalLink>
```

Comment:
Just out of curiosity what's the difference between `ExternalLink` and `DocLink`? :)

Reply:
`ExternalLink` takes `href` as an argument so it links anywhere. `<DocLink>` takes a path and appends it to the host: `https://developer.hashicorp.com` (formally vaultproject.io)

There's also `LearnLink` - but we no longer use the host `https://learn.hashicorp.com` as everything now lives at developer.hashicorp.com. 

However, the paths for `LearnLink` don't map 1:1 as they did for the DocLinks - so before we can remove the component and use doc-link everywhere, we have to update all of the paths. Right now there's a redirect in place that handles most of them so it's still working _for now_